### ISSUE TITLE
Stencil : is missing wrong dependency

### ISSUE THREAD

**Issue by DrMabuse23** [nxext/nx-extensions#701]
**Describe the bug**
after add @nxext/stencil to my nx monorepo i get the message 1
  ├── ✕ missing peer @nxext/vite@14.0.0

**To Reproduce**
use first step installation 

**Expected behavior**
vite should not be neccessary for stencil i guess

**Additional context**
<img width="1920" alt="Screenshot 2022-05-24 at 11 07 41" src="https://user-images.githubusercontent.com/797537/169996064-6dcc97b6-2610-4c26-96bb-9822479e4142.png">

its confusing that a stencil package needs any kind of vue packages maybe you could explain this please.

**Comment by DominikPieper** [comment]
Mh interesting, cause @nxext/vite isn't defined as a peer dep. 

Seems like pnpm reads the peerDeps of the peerDeps maybe?

`
  "peerDependencies": {
    "@nxext/svelte": "^14.0.0",
    "@nrwl/storybook": "^14.0.3",
    "@nrwl/cypress": "^14.0.3",
    "@nrwl/linter": "^14.0.3",
    "@nrwl/react": "^14.0.3",
    "@nrwl/angular": "^14.0.3",
    "@nrwl/jest": "^14.0.3",
    "@nrwl/workspace": "^14.0.3",
    "@stencil/core": "^2.15.1"
  },
  "peerDependenciesMeta": {
    "@nxext/svelte": {
      "optional": true
    },
    "@nrwl/storybook": {
      "optional": true
    },
    "@nrwl/react": {
      "optional": true
    },
    "@nrwl/angular": {
      "optional": true
    },
    "@nrwl/cypress": {
      "optional": true
    }
  },
`

That's all defined deps and the react/angular/svelte/vue packages just needed if you want to generate outputargets for these frameworks

**Comment by DrMabuse23** [comment]
Hey not sure i can follow. this is the package.json from the stencil package ? if yes why is there svelte react and agular needed ?

**Comment by DominikPieper** [comment]
It's from the @nxext/stencil package. There's a generator for the Stencil output targets for these framework. So they're just optional for if you use the generator for the output target bindings for these frameworks. Cause if you e.g. wan't to use the angular bindings it generates an angular lib for the Stencil lib for easier startup.

**Comment by DominikPieper** [comment]
If you don't use the output target generator you don't need these. I'm just a little confused as well cause I'm not sure how pnpm handles this. The listed packages like @nxext/vite and @nxext/vitest are dependencies from @nxext/svelte what's an optional dependency here for the Stencil package. 

The stencil package itself mentioned there will be installed with the first call from the app or lib generator

**Comment by DominikPieper** [comment]
Seems like a ppm thing cause I don't get them listed with yarn. ^^'

**Comment by DrMabuse23** [comment]
Yes i decided today starting with pnpm maybe  not the best idea...

**Comment by Jordan-Hall** [comment]
You can tend to ignore that pnpm error. Does it still work?

**Comment by DominikPieper** [comment]
Should't affect the usage at all (hopefully).

**Comment by DrMabuse23** [comment]
Ok yes and no i can`t usde the feature  --buildable and --publish for a library
and i can use my library in  my stencil app without imports 
<img width="1920" alt="Screenshot 2022-05-24 at 11 51 15" src="https://user-images.githubusercontent.com/797537/170004075-c5abf783-6a2f-4466-bb3e-3c3d5e686abb.png">

**Comment by DominikPieper** [comment]
What do you mean by you can't use them?

Wir können auch Deutsch schreiben wenn das einfacher ist :-)
